.github/workflows/progress-tracker.yml

```
@@ -0,0 +1,80 @@
+name: Progress Tracker
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - edited
+
+jobs:
+  track-progress:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v2
```

If we are going to run it on every edit (see above), then we should at least bomb out early instead of within the script:

```suggestion
      if: contains(github.event.issue.labels.*.name, env.FILTER_LABEL)
      - uses: actions/github-script@v2
```

---

Let's not go for running it on each issue update.

---

I have used this with some added condition for the manual trigger where the `event.issue` context will not be present.